<launch>

  <arg name="save_intermediate"           default="false" />
  <arg name="log_to_db"           	  default="false" />
  <arg name="update_metaroom"             default="true" />
  <arg name="machine"   		  default="localhost" />
  <!-- <arg name="use_NDT_registration"   default="true" /> -->
  <arg name="min_object_size"             default="500" />

  <arg name="user"   	default="" />
  <arg name="newest_dynamic_clusters" default="false" />

  <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)"/>

  <node machine="$(arg machine)" pkg="semantic_map" type="semantic_map_node" name="semantic_map_node" output="screen" respawn="true">
	<param name="save_intermediate"  type="bool" value="$(arg save_intermediate)"/>
	<param name="log_to_db"  type="bool" value="$(arg log_to_db)"/>
	<param name="update_metaroom"  type="bool" value="$(arg update_metaroom)"/>
	<param name="newest_dynamic_clusters"  type="bool" value="$(arg newest_dynamic_clusters)"/>
	<!-- <param name="use_NDT_registration"  type="bool" value="$(arg use_NDT_registration)"/> -->
	<param name="min_object_size"  type="int" value="$(arg min_object_size)"/>
  </node>

</launch>


